packages/local_storage/lib/src/local_storage.dart

```diff
@@ -26,51 +21,55 @@ class LocalStorage {
       version: 1,
       onCreate: (db, version) {
         db.execute('''
-          CREATE TABLE $_tasksTableName(
-            $_tasksIdColumnName INTEGER PRIMARY KEY,
-            $_tasksContentColumnName TEXT NOT NULL,
-            $_tasksStatusColumnName INTEGER NOT NULL
+          CREATE TABLE tasks(
+            id INTEGER PRIMARY KEY,
+            conctent TEXT NOT NULL,
+            status INTEGER NOT NULL
           )
           ''');
       },
     );
     return database;
   }
 
-  Future<void> addTask(String content) async {
+  /// Add new document to the table database
+  Future<void> addDocument({
+    required String collection,
+    required Map<String, dynamic> document,
+  }) async {
     final db = await database;
     await db.insert(
-      _tasksTableName,
-      {
-        _tasksContentColumnName: content,
-        _tasksStatusColumnName: 0, // domyślnie zadanie nie jest ukonczone
-      },
+      collection,
+      document,
     );
   }
 
   // Pobierz wszystkie zadania
-  Future<List<Map<String, dynamic>>> getTasks() async {
+  Future<List<Map<String, dynamic>>> getDocuments({
+    required String collection,
+  }) async {
     final db = await database;
-    return await db.query(_tasksTableName);
+    return await db.query(collection);
   }
 
   Future<void> editTask(int id, String newContent, int status) async {
     final db = await database;
     await db.update(
-      _tasksTableName,
+       'tasks',
       {
-        _tasksContentColumnName: newContent,
-        _tasksStatusColumnName: status,
+        'content': newContent,
+        'status' : status,
       },
-      where: '$_tasksIdColumnName = ?',
+      where: 'id = ?',
       whereArgs: [id],
     );
   }
-  Future <void> deleteTask(int id) async {
+
+  Future<void> deleteTask(int id) async {
     final db = await database;
     await db.delete(
-      _tasksTableName,
-      where: '$_tasksIdColumnName = ?',
+      'tasks',
+      where: 'id = ?',
       whereArgs: [id],
     );
   }
```

Comment:
again. We are in LocalStorage - we should not have any task related methods here - we should have it in TasksRepository instead.